README.md

```diff
@@ -200,3 +200,59 @@ You can run Celery workers for Concent with:
 ``` bash
 concent_api/celery worker --app concent_api --loglevel info --queues concent,conductor,verifier
 ```
+
+### Deploying GNTDeposit contract
+
+To be able to run e2e tests locally, that contains transactions on blockchain,
+Concent needs separate GNTDeposit contract to work without errors.
+
+1. Get [GNTDeposit.sol](https://github.com/golemfactory/golem-contracts/blob/master/contracts/GNTDeposit.sol)
+from `golem-contracts` repository.
+
+2. Fallow this [tutorial](https://medium.com/swlh/deploy-smart-contracts-on-ropsten-testnet-through-ethereum-remix-233cd1494b4b)
+to deploy GNTDeposit.sol on `rinkeby` ethereum testnet.
+
+    It involves installing `MetaMask` on your browser, creating ethereum account with some testnet ETH and copy
+    GNTDeposit.sol with all dependencies to single file in solidity IDE - `remix`. You can use
+    [solidity-flattener](https://github.com/BlockCatIO/solidity-flattener) to do so.
+
+3. There are four variables needed to deploy new GNTDeposit contract:
+
+    - `_token` - GNTB contract address. For all testnet environments, it should be hardcoded to
+    `0x123438d379BAbD07134d1d4d7dFa0BCbd56ca3F3`.
+
+    - `_concent` - Concent ethereum address. It can be the same for all testnet environments
+    `0xa5f3cd35385755e64b47f20a5d7334d7013f8015`.
+
+    - `_coldwallet` - Another or the same concent ethereum address. For locale environment I use address I'm logged in
+    to in `MetaMask`.
+
+    - `_withdrawal_delay` - Says for how long (in seconds) deposits are being blocked. We used mainnet value so far,
+    which is 48h (172800).
+
+4. After successfully creating GNTDeposit contract, we can add some `local_settings`:
+
+    - `GNT_DEPOSIT_CONTRACT_ADDRESS`
+
+    - `CONCENT_ETHEREUM_PRIVATE_KEY` in bytes and `CONCENT_ETHEREUM_PUBLIC_KEY` in hex of our `_coldwallet`.
+
+5. Additionally we might need to give some deposits to concent ethereum address as well as for provider and requestor
+that are involved in e2e tests. Clear concent databases. Run `python manage.py shell` :
+
+``` python
+from core.payments.payment_interface import PaymentInterface
+
+p = PaymentInterface()  # eth address for "p" depend on `CONCENT_ETHEREUM_PRIVATE_KEY` and `CONCENT_ETHEREUM_PUBLIC_KEY`
+p.request_gnt_from_faucet()
+# check if You have GNT with `p.get_gnt_balance(p.get_eth_address())`
+
+p.open_gate()
+# check if command work with `p.get_gate_address()` (should return eth address)
```

Comment:
`command work` -> `the command above worked`

Reply:
Done